core/scripts/cli/sponsor/redeem.js

```diff
@@ -12,15 +12,7 @@ const redeem = async (web3, artifacts, emp) => {
   const token = await SyntheticToken.at(tokenAddress);
   const walletTokens = await token.balanceOf(sponsorAddress);
 
-  console.log("You have:");
-  console.log(
-    "Position: " +
-      fromWei(collateral) +
-      " WETH backing " +
-      fromWei(position.tokensOutstanding.toString()) +
-      " synthetic tokens"
-  );
-  console.log("Wallet: " + fromWei(walletTokens) + " synthetic tokens");
+  console.log("Your wallet has: " + fromWei(walletTokens) + " synthetic tokens");
```

Comment:
Why show this again? Just curious

Reply:
I don't we've shown this information anywhere, yet. This value isn't the number of tokens in the Position, it's the number of tokens in the sponsor's wallet, which they could potentially redeem.